jobsdb/jobsdb.go

```diff
@@ -2557,7 +2558,7 @@ func (jd *Handle) refreshDSList(ctx context.Context) error {
 	start := time.Now()
 	var err error
 	defer func() {
-		stats.Default.NewTaggedStat("refresh_ds_loop", stats.TimerType, stats.Tags{"customVal": jd.tablePrefix, "error": strconv.FormatBool(err != nil)}).Since(start)
+		stats.Default.NewTaggedStat("refresh_ds_loop", stats.TimerType, stats.Tags{constants.CustomVal: jd.tablePrefix, "error": strconv.FormatBool(err != nil)}).Since(start)
```

Comment:
Are we going to use the same set of constants for covering both log fields & stat tags?

Reply:
I think it's convenient, might need some changes like you've pointed out in another comment.

Reply:
cleared this one out too.